Azure/sonic-buildimage
# [sonic-platform-common] Enable unit tests during build for Python 3 package

**- Why I did it**

To enable running Pytest unit tests before building the Python 3 sonic-platform-common package

**- How I did it**

- Add Python 3 sonic-config-engine package as a dependency of Python 3 sonic-platform-common package (needed for both runtime and unit tests)
- No longer disable unit tests when building Python 3 sonic-platform-common package

**- How to verify it**

Build a SONiC image. Check sonic-platform-common Python 3 package build log and confirm unit tests were run

**- Which release branch to backport (provide reason below if selected)**
